assets/templates/partials/header.tmpl

```diff
@@ -57,191 +59,251 @@
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/educationandchildcare">{{ localise "EducationChildcare" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/elections">{{ localise "Elections" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/healthandsocialcare">{{ localise "HealthSocialCare" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/householdcharacteristics">{{ localise "HouseholdCharacteristics" .Language 1 }}</a>
+                    {{if not .SearchDisabled}}
+                        <li class="nav--controls__item ">
+                            <a href="#nav-search" id="search-toggle" aria-controls="nav-search" aria-expanded="false" class="nav--controls__search">
+                                <span class="nav--controls__text">{{ localise "Search" .Language 1 }}</span>
+                            </a>
                         </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/housing">{{ localise "Housing" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/leisureandtourism">{{ localise "LeisureTourism" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/personalandhouseholdfinances">{{ localise "PersonalHouseholdFinances" .Language 1 }}</a>
-                        </li>
-                        <li class="primary-nav__child-item  js-expandable__child">
-                            <a class="primary-nav__child-link" tabindex="-1" href="/peoplepopulationandcommunity/populationandmigration">{{ localise "PopulationMigration" .Language 1 }}</a>
+                    {{end}}
+                </ul>
+                <ul class="wrapper nav-main--hidden primary-nav__list" id="nav-primary" aria-expanded="false">
+                    <li class="primary-nav__item {{if eq $uri " /"}}primary-nav__item--active{{end}} js-nav hide--sm old-ie--display-block">
```

Comment:
It doesn't look like a deeper dot context here (e.g. not a range or anything like that) so `.URI` should be fine and the `$uri` definition can be removed